modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/containers/PaginationControls/PaginationControls.soy

```diff
@@ -3,10 +3,15 @@
 {template .render}
 	{@param activePage: number}
 	{@param showSubmitButton: bool}
+	{@param submitLabel: string}
 	{@param total: number}
 	{@param? _handleNextClicked: any}
 	{@param? _handlePreviousClicked: any}
 	{@param? readOnly: bool}
+	{@param? strings: map<string, string>}
+
+	{let $currentNextLabel: $strings != null ? $strings['next'] : 'next' /}
```

Comment:
Hi @mclomello, just a question. I'm not sure if I'm aware of the whole case, but is there a possibility of strings come null? If so, the value of currentNextLabel would be a not formated string and always in English. Is this expected?